charts/agent-k8s/values.yaml

```diff
@@ -59,6 +59,10 @@ fullnameOverride: ""
 controllerNodeSelector: {}
 # -- Kubernetes Node Selector for assigning worker agents and scheduling agent tasks to specific nodes in the cluster.
 # The selector must match a node's labels for the pod to be scheduled on that node.
+controllerTolerations: {}
```

Comment:
@yurii-kryvosheia pod's tolerrations field is expected to be an array - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#pod-v1-core